wolfSSL/src/chapter02.md

```diff
@@ -472,6 +588,54 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
+#### WOLF_PRIVATE_KEY_ID
+
+FIPS v5 and older doesn't support WOLF_PRIVATE_KEY_ID with PK callbacks. Required for client authentification in some cases.
+
+#### WOLFSSL_WOLFSENTRY_HOOKS
+
+Enables wolfSentry hook support.
+
+#### WOLFSSL_CERT_EXT
+
+Cert exit, key and cert generation feature.
+
+#### WOLFSSL_CERT_REQ
+
+Cert requirement, key, and cert generation feature.
+
+#### WOLFSSL_SSLKEYLOGFILE
+
+Warn if secrets logging is enabled. The SHOW_SECRETS and WOLFSSL_SSLKEYLOGFILE options should only be used for debugging and never in a production environment.
+
+#### WOLFSSL_SSLKEYLOGFILE_OUTPUT
+
+It can be used as an Optional Pre-Master-Secret logging for Wireshark.
+
+#### WOLFSSL_HAVE_CERT_SERVICE
+
+Used by autoconf to see if cert service is available.
+
+#### WOLFSSL_HAVE_WOLFSCEP
+
+Enable feature used by autoconf to see if wolfSCEP is available.
+
+#### WOLFSSL_HAVE_MIN
+
+Needed for use with older versions of xc16.
+
+#### WOLFSSL_HAVE_TLS_UNIQUE
+
+Added in libest port: allow applications to get the 'tls-unique' Channel Binding Type (https://tools.ietf.org/html/rfc5929#section-3). This is used in the EST protocol to bind an enrollment to a TLS session through 'proof-of-possession' (https://tools.ietf.org/html/rfc7030#section-3.4 and https://tools.ietf.org/html/rfc7030#section-3.5).
+
+#### WOLFSSL_ENCRYPTED_KEYS
+
+Enable for encrypted keys PKCS8 support.
+
+#### WOLFSSL_CUSTOM_OID
+
+Key Generation feature. Enables custom OID support for subject and request extensions.
```

Comment:
"Certificate feature" This also applies to parsing certificates with custom OID's. Not just certificate generation.